Description: Missing part of the arm64 patch (upstream fix after the 3.0.0rc1
release)
Source: http://svn.freepascal.org/cgi-bin/viewvc.cgi/trunk/compiler/systems.pas?r1=32102&r2=32101&view=patch
Bug-Debian: https://bugs.debian.org/807479
Index: fpc/fpcsrc/compiler/systems.pas
===================================================================
--- fpc.orig/fpcsrc/compiler/systems.pas
+++ fpc/fpcsrc/compiler/systems.pas
@@ -338,7 +338,7 @@ interface
 
        { all systems where a value parameter passed by reference must be copied
          on the caller side rather than on the callee side }
-       systems_caller_copy_addr_value_para = [system_aarch64_darwin];
+       systems_caller_copy_addr_value_para = [system_aarch64_darwin,system_aarch64_linux];
 
        { pointer checking (requires special code in FPC_CHECKPOINTER,
          and can never work for libc-based targets or any other program
